beta-src/src/components/ui/WDDeleteReminder.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
```

Comment:
Is WDDeleteReminder the best name for this component? I see it's making the notifications for destroys now but the plan is to use this logic for all game notifications in the future.